android/java/res_template/xml/file_paths.xml

```diff
@@ -8,6 +8,7 @@
 <!-- for the ContentProvider. -->
 
 <paths xmlns:android="http://schemas.android.com/apk/res/android">
+    <root-path name="root" path="." />
```

Comment:
Is there any downside to removing this? It seems like it was needed in order to open downloaded files.

Reply:
Oh wait, it's being added, nevermind :-)